config/routes.rb

```diff
@@ -1,5 +1,5 @@
 Rails.application.routes.draw do
-  resources :families, only: [:index, :show, :new, :create, :destroy]
+  resources :families, only: [:index, :show, :new, :create, :update, :edit, :destroy]
```

Comment:
Since you have all the actions now, you can remove the `:only` option and just use:
`resources :families`

Reply:
Ok! Thanks!
Done with fixup in 9308a00